Helper/Config.php

```diff
@@ -299,6 +299,12 @@ public function setStoreMetadata(?int $storeId, ProtectMetadata $metadata): void
         $this->flushCaches();
     }
 
+    /**
+     * Deletes the store metadata from the magento config
+     * @param int $storeId Id of the store to remove
+     * @return void
+     */
+
```

Comment:
Uhhh

Reply:
yeah, I realized what I did,  I went through and did the nulling changes, then changed my mind and through I should make a token delete command, realized that was going to get more complicated than the command should have been undid those changes, and just made a mess of things. 

I'm not at my best in the morning